casbin/go.mod

```diff
@@ -1,23 +1,28 @@
 module github.com/gofiber/contrib/casbin
```

Comment:
its a breaking change pls add a new version suffix

github.com/gofiber/contrib/casbin/v2

Reply:
Thanks a lot for the reminder - I missed it. I will make a commit soon.